pkg/sync/syncer.go

```diff
@@ -1801,6 +2353,18 @@ func WithSkipFullSync() SyncOpt {
 	}
 }
 
+func WithExternalResourceC1ZPath(path string) SyncOpt {
+	return func(s *syncer) {
+		s.externalResourceC1ZPath = path
+	}
+}
+
+func WithExternalResourceEntitlementIdFilter(entitlementId string) SyncOpt {
```

Comment:
What if you care about two entitlements?

Reply:
like to include users/groups that have grants for any of these entitlements?